GitUI/ScriptsEngine/ScriptOptionsParser.cs

```diff
@@ -283,12 +281,12 @@ private static string GetRemotePath(string url)
             return "";
         }
 
-        private static string? ParseScriptArguments(string arguments, string option, IWin32Window owner,
-            IScriptOptionsProvider? scriptOptionsProvider, IGitUICommands uiCommands, IReadOnlyList<GitRevision> allSelectedRevisions,
-            in IList<IGitRef> selectedTags, in IList<IGitRef> selectedBranches, in IList<IGitRef> selectedLocalBranches,
-            in IList<IGitRef> selectedRemoteBranches, in IList<string> selectedRemotes, GitRevision selectedRevision,
-            in IList<IGitRef> currentTags, in IList<IGitRef> currentBranches, in IList<IGitRef> currentLocalBranches,
-            in IList<IGitRef> currentRemoteBranches, GitRevision currentRevision, string currentRemote)
+        private static string? ParseScriptArguments(string arguments, string option, IWin32Window owner, IScriptOptionsProvider? scriptOptionsProvider,
+            IGitUICommands uiCommands, IReadOnlyList<GitRevision> allSelectedRevisions, in IList<IGitRef> selectedTags,
+            in IList<IGitRef> selectedBranches, in IList<IGitRef> selectedLocalBranches, in IList<IGitRef> selectedRemoteBranches,
+            in IList<string> selectedRemotes, GitRevision selectedRevision, in IList<IGitRef> currentTags,
+            in IList<IGitRef> currentBranches, in IList<IGitRef> currentLocalBranches, in IList<IGitRef> currentRemoteBranches,
+            GitRevision currentRevision, string currentRemote)
```

Comment:
This is effectively the same code. The change does not improve the readability.
If you really want to change it, then one argument per line.

Reply:
I still think we should use a struct or a class here.

Reply:
Reverted (like all the below changes). Sorry, I have used the refactoring feature of VS without seeing that it changed all the arguments formatting.
I have squashed everything to remove these changes made in one commit and reverted in a later one and now history's much cleaner.